### PR TITLE
Hotfix/cobalt

### PR DESCRIPTION
**Pull request by andre-merzky**
revive cobalt support - tested (within bounds)  on Theta

### CODE REVIEW

**Comment by andre-merzky** [comment]
@iparask : maybe you have the time to review this one?

File: src/radical/saga/adaptors/slurm/slurm_job.py [hunk 1]
```diff
@@ -404,6 +404,9 @@ def _open(self):
         elif 'frontera' in self.rm.host.lower():
             self._ppn = 56
 
+        elif 'comet' in self.rm.host.lower():
```
Comment (iparask) [inline src/radical/saga/adaptors/slurm/slurm_job.py:407]: Why is there a change on slurm? This is not cobalt relevant?

Comment (andre-merzky) [reply src/radical/saga/adaptors/slurm/slurm_job.py:407]: This is because of an out-of-sync merge, will be resolved in devel.  Thanks for noticing.

**Comment by andre-merzky** [comment]
Thanks @iparask, and fair enough on cobalt access.  I decided to merge this anyway, as we tested during the covid effort, and it's an isolated code change (limited to the adaptor code).

**Pull request merged**